docs/en/user_guides/train_test.md

```diff
@@ -0,0 +1,350 @@
+# train_test
+
+## Prepare dataset for training and testing
+
+This section details how to prepare the dataset for MMGeneration and provides a standard way which we have used in our default configs. We recommend that all of the users may follow the following steps to organize their datasets.
+
+### Datasets for unconditional models
+
+It's much easier to prepare dataset for unconditional models. Firstly, please make a directory, named `data`, in the MMGeneration project. After that, all of datasets can be used by adopting the technology of symlink (soft link).
```

Comment:
remove unnecessary sentences: 'It's much easier to prepare dataset for unconditional models.'